src/pages/SignIn/SignIn.jsx

```diff
@@ -0,0 +1,134 @@
+import { Button, TextField, Typography } from "@mui/material";
+import { Box } from "@mui/system";
+import { Field, Formik } from "formik";
+import { object, string } from "yup";
+import { Link } from "react-router-dom";
+import axios from "axios";
+import { useDispatch } from "react-redux";
+import { useHistory } from "react-router-dom";
+
+import Header from "components/Header/Header";
+import { ROUTES } from "config/constants";
+
+import { add } from "../../utils/redux/features/addUser/userSlice";
+
+import {
+  FormBox,
+  StyledBox,
+  StyledForm,
+  TypographyButton,
+  StyledTypography,
+  SignInHeader,
+} from "./styled";
+
+const loginValues = {
+  email: "",
+  password: "",
+};
+
+function utf8_to_b64(str) {
+  return window.btoa(unescape(encodeURIComponent(str)));
+}
+
+const SignIn = () => {
+  const dispatch = useDispatch();
+  const history = useHistory();
+  const redirectToHomePage = () => {
+    history.push("/notes");
+  };
+
+  const onSubmit = (values) => {
+    const session_url =
+      "https://note-app-training-server.herokuapp.com/api/users/auth";
```

Comment:
Move to constants api routes (`users/auth`)
And move server link(`https://note-app-training-server.herokuapp.com/api/`) to the .env file